ReactWindows/ReactNative/UIManager/UIViewOperationQueue.cs

```diff
@@ -183,7 +223,7 @@ public Task RemoveRootViewAsync(int rootViewTag)
                 if (pair.Value.rootViewCount == 0)
                 {
                     // We can remove this queue and then destroy
-                    _dispatcherToOperationQueueInfo.Remove(pair.Key);
+                    _dispatcherToOperationQueueInfo.TryRemove(pair.Key, out _);
```

Comment:
Do we care if something was removed twice (i.e., should we check the return value of TryRemove?

Reply:
I added checks for duplicate adds opportunistically, the AddOrUpdate begged for it.
Here though I would have to add a new if, too noisy. Will never happen ever, anyway.